kuma/revision/urls.py

```diff
@@ -0,0 +1,14 @@
+from django.conf.urls import url
+
+from . import views
+
+
+urlpatterns = [
+    # Serve the revision hashes.
+    url(r'^media/revision.txt$',
```

Comment:
This is a better place than "misc". I'm a little worried it will get confused with wiki revisions, but we can move code where ever. Maybe we can do a sorting exercise at an all hands to discover a better grouping. It seems to be there are some meta views like this and the health checks that could be grouped together under "endpoints that describe the current deployment". No change needed now.

Reply:
Oh, that's a good point. Sounds good.